build/compiler.py

```diff
@@ -357,6 +357,12 @@ def lint(self, fix=False, force=False):
     deps = self.source_files + [self.config_path]
     if not force and not _must_build(self.output, deps):
       return True
+    """Windows shows an error when the file location has '\\' so for 
```

Comment:
I just noticed this isn't the function docstring.  This should just be a comment.  Also you don't need to explain what the code is doing, just why it is.  So this would be better.

```py
# Windows shows an error when the file location has '\\'.
```